contrib/oasis.service

```diff
@@ -1,12 +1,16 @@
+# cp oasis.service ~/.config/systemd/user
```

Comment:
What do you think about `mkdir -p ~/.config/systemd/user/` above this line? I had `~/.config` but I'm on a fresh Arch install that didn't have `~/.config/systemd`.

Reply:
will add that as well.